### ISSUE TITLE
Error with chimera.vsearch (version 1.43)

### ISSUE THREAD

**Issue by Puumanamana** [mothur/mothur#675]
Hi,

I am currently using the latest version of Mothur (v 1.43) as part of a pipeline. I was able to use it for one dataset, but for another one, I got an error on the command `chimera.vsearch(fasta=current, count=current, dereplicate=t)`
When I use version 1.43 as well as 1.42.3, I get a segfault:
```
should never get here, quitting
[1]    97754 segmentation fault (core dumped)
```
However, it seems to run fine with version 1.41.3
Any help would be appreciated! 
Let me know if you need any file to reproduce the error.

**Comment by mothur-westcott** [comment]
Do you have any '-' characters in your sample names? The first thing the chimera.vsearch command does is divide the dataset by sample for denovo processing. The split.group command does the splitting in our latest versions. It accepts a string with the list of groups separated by '-' characters. If your sample names contain '-' characters then the split.group command does not find all the groups and this error message is outputted.

**Comment by Puumanamana** [comment]
No, all sample names are alphanumeric or underscores. After looking a bit more, I found that one sample was empty. When I remove it, it works.
However, this empty sample was generated after runinng `align.seqs()` followed by `screen.seqs(fasta=current, count=current, optimize=start-end, criteria=95)`, which discarded all empty samples except one.

**Comment by mothur-westcott** [comment]
Could you send the input files for the screen.seqs command to mothur.bugs@gmail.com so I can take a closer look?

**Comment by mothur-westcott** [comment]
Thank you for reporting this issue and helping me to track it down. The remove.seqs command which is run by screen.seqs was not removing groups that contained 0 sequences. I have fixed the issue and the change will be part of our next release. In the meantime, you can work around the issue by running the following:

mothur > screen.seqs(fasta=all_esv.filter.fasta, count=all_esv.count_table, minlength=50)
mothur > make.table(count=current) - the reading of the table here removes the 0'd groups
...

Kindly,
  Sarah
